cmd/gateway-main.go

```diff
@@ -47,6 +47,10 @@ EXAMPLES:
 
   2. Start minio gateway server bound to a specific ADDRESS:PORT.
       $ {{.HelpName}} --address 192.168.1.101:9000 azure
+
+  3. Gateway server connecting to a specific Azure Blob Storage endpoint.
+      $ {{.HelpName}} azure https://storage.minio.io
```

Comment:
`core.windows.net` is the default one. But for "Azure stack" deployments (https://azure.microsoft.com/en-us/overview/azure-stack/) endpoint would be custom http://azure.endpoint.com

Probably random IP address is better than storage.minio.io so that users don't try storage.minio.io on command line.